### ISSUE TITLE
Difference between ETP Strict and ETP Custom with all 3rd party cookies blocked

### ISSUE THREAD

**Issue by GlassGruber** [arkenfox/user.js#1521]
Hei there I've update from ESR 91 to 102 and was checking if everything is in place and I remembered the pinned warning about https://github.com/arkenfox/user.js/issues/1334.

So I double checked my current situation and looked around a little and noticed in privacy settings that I'm using ETP custom with all 3rd party cookies blocked. 
AFAICT I've set up this long time ago via GUI (not user.js) and never really had problems anywhere.

Now I've checked and this will set `network.cookie.cookieBehavior` to 1 instead of 5.
IIRC this setting was also involved to enable FPI/dFPI some versions ago but is now deprecated in favor of network partitioning (right?).

So basically the question is:
am I messing up something else here with `network.cookie.cookieBehavior` =1 instead of =5? 
Is it just blocking all 3rd party cookies so I'm _stricter_ than strict? 👀 

(Now I'll hide and stand perfectly still hoping that Tyrannosaurus-Pants won't notice me while triggered into roaring rage by ETP Custom in title 🚽🛖🦖)

**Comment by rusty-snake** [comment]
> Now I've checked and this will set `network.cookie.cookieBehavior` to 1 instead of 5.
> IIRC this setting was also involved to enable FPI/dFPI some versions ago but is now deprecated in favor of network partitioning (right?).

No. Network partitioning and dFPI are two things.

https://developer.mozilla.org/en-US/docs/Web/Privacy/State_Partitioning

**Comment by GlassGruber** [comment]
Thank you rusty! 👍 
It's incredible how many useful resources Mozilla is putting out, unfortunately they are spread on so many websites, I use MDN quite often and never noticed there is a [Privacy section](https://developer.mozilla.org/en-US/docs/Web/Privacy)!

This is kinda answering what I'm asking in section [Past approaches](https://developer.mozilla.org/en-US/docs/Web/Privacy/State_Partitioning#past_approaches_to_blocking_cross-site_tracking)
> For example, our "block all third-party cookies" policy will prevent all domains from accessing certain storage APIs when loaded in a third-party context. Our current [default cookie policy](https://developer.mozilla.org/en-US/docs/Web/Privacy/Storage_Access_Policy) blocks access in a third-party context only for domains classified as trackers.

I'm still a bit confused that this is considered a "past" approach. Is still valid? 

If it's valid I guess I can leave things as they are ATM right?

**Comment by rusty-snake** [comment]
The "Our current default cookie policy blocks access … only for … trackers" is likely not valid any more since 103 (which is one week old).

Whether `cookieBehavior=1` blocks all APIs which are isolated with `cookieBehavior=5` is something I can not tell you but my guess would be that there wasn't any real development for `cookieBehavior=1` for years.

**Comment by GlassGruber** [comment]
I see, I've tried some search-fu around for `cookieBehavior=1` but nothing relevant came out. 
I've tried also with [searchfox ](https://searchfox.org/mozilla-central/search?q=network.cookie.cookieBehavior&path=&case=false&regexp=false)but I'm not very expert in where I should look at, results look a bit cryptic to me.

Also I'm wondering if there is not much interest with block everything because the behavior of blocking 3rd party is more telling entropy wise than just let them through in isolated containers?

Anyway I'm switching to 5 (just in case, maybe 🦖 won't smell me with my new perfume _cookieBehaviour N°5_)

**Comment by Thorin-Oakenpants** [comment]
good lord. If behavior is not 5 then you lose a fuckload of partitioning - SHAME on all of you - namely everything that isn't under network partitioning. So if you need to relax a third party, boom, no fucking protection. Use uBO more (e.g. I block 3rd party by default which is in effect the same as using 1 because I never connect to 4rd parties to start with) - which also gives you perf benefits

The whole point of 5 is that everything works by default, no borked sites - and it's partitioned. This is BETTER than 1. 1 breaks shit. 5 doesn't break shit. 5 is partitioned, no tracking and is no more of a "privacy/tracking" risk than 1. 1 is fucking stupid - it is not MORE PRIVATE/STRICT (you still connect to the third parties). If you want MORE PRIVATE/STRICT then ramp up uBO (like Pants. Pants is smart, be like Pants)

marking as invalid out of spite because of my pinned labeled announcement not to fucking discuss anything other than ETP strict
